src/main/java/com/btkelly/gnag/reporters/BaseReporter.java

```diff
@@ -95,8 +99,22 @@ private T getReport(Project project) throws JAXBException {
         }
     }
 
-    private java.io.File getReportFile(Project project) {
-        return new java.io.File(project.getProjectDir(), getReportFilePath());
+    @Nullable
+    private java.io.File getReportFile(Project project) throws FileNotFoundException {
```

Comment:
OTOH, having it not fail on exception is what allows gnag to still work with Java-only projects for now...